### PR TITLE
Linear Regression - Documentation

### PR DESCRIPTION
**Pull request by aanchalsethi**

### CODE REVIEW

File: Linear Regression/README.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+Regression:
```
Comment (ashwani-rathee) [inline Linear Regression/README.md:1]: Just update with proper headings :)
```suggestion
 ## Regression:
```

Comment (ashwani-rathee) [reply Linear Regression/README.md:1]: Do it with all the headings

**Review by ashwani-rathee — COMMENTED** [review]
Other than this minor issue,it's ready to merge imo @aanchalsethi

File: Linear Regression/README.md (hunk 1 above)
Comment (aanchalsethi) [reply Linear Regression/README.md:1]: okay sir

**Comment by ashwani-rathee** [comment]
@aanchalsethi please add references in end for further reading and some links to yt vids...after that it looks good to me, @geekquad what do you think?

**Comment by aanchalsethi** [comment]
Yes sir i have added a link to yt video .

**Review by geekquad — APPROVED** [review]

**Pull request merged**